src/module.rs

```diff
@@ -215,6 +215,12 @@ impl ModuleInstance {
         self.globals.borrow_mut().push(global)
     }
 
+    /// Access all globals. This is a non-standard API so it's unlikely to be
+    /// portable to other engines.
+    pub fn globals<'a>(&self) -> &[GlobalRef] {
+        &*self.globals.borrow()
```

Comment:
I wonder how does this work: is it acctually possible to return a reference returned from `borrow`?

Reply:
arg I pushed broken code, I've corrected this to return `cell::Ref`.